assets/onboarding/features/index.jsx

```diff
@@ -0,0 +1,166 @@
+		id: 'certificates',
+		title: __( 'Certificates', 'sensei-lms' ),
+		description: __(
+			'Award your students with a certificate of completion and a sense of accomplishment after finishing a course.',
+			'sensei-lms'
+		),
+		learnMoreLink: 'https://woocommerce.com/products/sensei-certificates/',
+		errorMessage: __(
+			'Error message here, maecenas faucibus mollis interdum tristique euismod.',
+			'sensei-lms'
+		),
+		status: ERROR_STATUS,
+	},
+	{
+		id: 'media_attachments',
+		title: __( 'Media Attachments', 'sensei-lms' ),
+		description: __(
+			'Provide your students with easy access to additional learning materials, from audio files to slideshows and PDFs.',
+			'sensei-lms'
+		),
+		learnMoreLink:
+			'https://woocommerce.com/products/sensei-media-attachments/',
+		status: SUCCESS_STATUS,
+	},
+	{
+		id: 'content_drip',
+		title: __( 'Content Drip', 'sensei-lms' ),
+		price: '$29.00 per year',
+		description: __(
+			'Keep students engaged and improve knowledge retention by setting a delivery schedule for course content.',
+			'sensei-lms'
+		),
+		learnMoreLink: 'https://woocommerce.com/products/sensei-content-drip/',
+		status: LOADING_STATUS,
+	},
+].map( ( feature ) => ( {
+	...feature,
+	title: `${ feature.title } — ${
+		feature.price ? feature.price : __( 'Free', 'sensei-lms' )
+	}`,
+} ) );
+
+/**
+ * Features step for setup wizard.
+ */
+const Features = () => {
+	const [ confirmationActive, toggleConfirmation ] = useState( false );
+	const [ feedbackActive, toggleFeedback ] = useState( false );
+	const [ selectedFeatureIds, setSelectedFeatureIds ] = useState( [] );
+	const { goTo } = useQueryStringRouter();
+
+	const getSelectedFeatures = () =>
+		features.filter( ( feature ) =>
+			selectedFeatureIds.includes( feature.id )
+		);
+
+	const finishSelection = () => {
+		if ( 0 === selectedFeatureIds.length ) {
+			goToNextStep();
+			return;
+		}
+
+		toggleConfirmation( true );
+	};
+
+	const goToInstallation = () => {
+		toggleConfirmation( false );
+		toggleFeedback( true );
+	};
+
+	const goBackToSelection = () => {
+		toggleFeedback( false );
+	};
+
+	const goToNextStep = () => {
+		goTo( 'ready' );
+	};
+
+	return (
+		<>
+			<div className="sensei-onboarding__title">
+				<H>
+					{ __(
+						'Enhance your online courses with these optional features!',
+						'sensei-lms'
+					) }
+				</H>
+			</div>
+			<Card className="sensei-onboarding__card">
+				{ feedbackActive ? (
+					<InstallationFeedback
+						features={ getSelectedFeatures() }
+						onContinue={ goToNextStep }
+					/>
+				) : (
+					<FeaturesSelection
+						features={ features }
+						selectedIds={ selectedFeatureIds }
+						onChange={ setSelectedFeatureIds }
+						onContinue={ finishSelection }
+					/>
+				) }
+			</Card>
+
+			{ feedbackActive && (
+				<div className="sensei-onboarding__bottom-actions">
+					<Button isTertiary onClick={ goBackToSelection }>
+						&larr;&nbsp;
+						{ __( 'Back to optional features', 'sensei-lms' ) }
```

Comment:
Where did this bit come from? I don't think we want to add this.

Reply:
Perfect! Removed here: https://github.com/Automattic/sensei/pull/3121/commits/40be4b91582f49e3c1df31baa524d43a850385bb

Taking the opportunity to ask you a question. If the user already has some of the features installed, we allow them to select and we just skip these installations?

Reply:
I really can't see this happening too often since Sensei is required in order to run any Sensei extension, but if they _do_ happen to have one more more installed, then it's probably best if we don't make those selectable (i.e. disable the checkbox). I would say hide them altogether, but then if they have them _all_ installed, this would be a blank page.